classes/Hook/HookActionValidateOrder.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\Module\Ps_Googleanalytics\Hooks;
+
+use Context;
+use Ps_Googleanalytics;
+
+class HookActionValidateOrder implements HookInterface
+{
+    /**
+     * @var Ps_Googleanalytics
+     */
+    private $module;
+
+    /**
+     * @var Context
+     */
+    private $context;
+
+    /**
+     * @var array
+     */
+    private $params;
+
+    public function __construct(Ps_Googleanalytics $module, Context $context)
+    {
+        $this->module = $module;
+        $this->context = $context;
+    }
+
+    /**
+     * run
+     *
+     * @return void
+     */
+    public function run()
+    {
+        // Check if we are creating backoffice order
+        if ($this->context->controller->controller_name != 'AdminOrders' && $this->context->controller->controller_name != 'Admin') {
```

Comment:
@Hlavtox 

hey Daniel

you sure you wanted to do `$this->context->controller->controller_name != 'Admin'`? when's situation where the controller is `Admin`? 🤔 maybe you wanted to use `strpos` there?

Reply:
Can't remember after 3 months, I need to check it again

Reply:
@kpodemski Yes it's correct.

`actionValidateOrder` - `dump($this->context->controller);`

Dies on this URL: http://localhost/81x/admin-dev/index.php/sell/orders/place?_token=dkwJoC-fWxJi0Qr5YgeSYtTvGmT1OOPEi4JbJqqDbjQ

![Snímek obrazovky 2023-10-30 172347](https://github.com/PrestaShop/ps_googleanalytics/assets/6097524/89235ea2-c496-4982-adc6-7edf8af49f5c)